src/main/java/com/google/api/codegen/util/ProtoParser.java

```diff
@@ -148,6 +152,11 @@ public ResourceDescriptor getResourceDescriptor(MessageType element) {
     return getProtoExtension(element, ResourceProto.resource);
   }
 
+  public List<ResourceDescriptor> getFileLevelResourceDescriptors(ProtoFile protoFile) {
```

Comment:
It looks like this method is called only from another method in this class. Is there a reason to have it as `public`?

Reply:
Nope. Too happy copying and pasting.